cmd/bnsd/x/username/model.go

```diff
@@ -65,53 +76,53 @@ func (t *Token) Copy() orm.CloneableData {
 	}
 }
 
-// NewTokenBucket returns a ModelBucket instance limited to interacting with a
-// Token model only.
-// Only a valid Username instance should be used as a key. Alternatively tokens can
-// be queried by owner.
+// NewTokenBucket returns a ModelBucket instance limited to interacting with
+// the Token model only.
+// Only a valid username (<name>*<label>) should be used as a key.
```

Comment:
An index for the namespace may become handy if you have to transfer a namespace which includes transfer of all usernames or other namespace triggered operations.